install_config/cluster_metrics.adoc

```diff
@@ -289,6 +289,12 @@ ifdef::openshift-origin[]
 /usr/share/openshift/examples/infrastructure-templates/origin/metrics-deployer.yaml
 ----
 ====
+
+[NOTE]
+Depending on your installation method, the template may not be present in your OpenShift installation. If so, the template can be found at the following GitHub location:
```

Comment:
News to me, if not. Reasoning is so that it's easily readable in source (doesn't scroll off horizontally) and isn't dependent on whether the text viewer at the time soft wraps. Also makes easier spotting of diffs and commenting on individual lines.